sql/moz-fx-data-shared-prod/addon_moderations_derived/cinder_decisions_raw_v1/query.py

```diff
@@ -105,6 +91,21 @@ def upload_to_bigquery(data, project, dataset, table_name, date):
             type_=bigquery.TimePartitioningType.DAY,
             field="date",
         ),
+        schema=[
+            bigquery.SchemaField("date", "DATE"),
+            bigquery.SchemaField("decision_type", "STRING", mode="NULLABLE"),
+            bigquery.SchemaField("user", "STRING", mode="NULLABLE"),
+            bigquery.SchemaField("created_at", "STRING", mode="NULLABLE"),
+            bigquery.SchemaField("entity_id", "STRING", mode="NULLABLE"),
+            bigquery.SchemaField("entity", "STRING", mode="NULLABLE"),
+            bigquery.SchemaField("uuid", "STRING", mode="NULLABLE"),
+            bigquery.SchemaField("entity_slug", "STRING", mode="NULLABLE"),
+            bigquery.SchemaField("job_id", "STRING", mode="NULLABLE"),
+            bigquery.SchemaField("job_assigned_at", "STRING", mode="NULLABLE"),
+            bigquery.SchemaField("queue_slug", "STRING", mode="NULLABLE"),
+            bigquery.SchemaField("typed_metadata", "STRING", mode="NULLABLE"),
+            bigquery.SchemaField("applied_policies", "STRING", mode="NULLABLE"),
+        ],
```

Comment:
To avoid having to maintain the schema both here and in `schema.yaml` you could just load it from `schema.yaml`:
```suggestion
        schema=Schema.from_schema_file(schema_file_path).to_bigquery_schema(),
```
That would require first importing `Schema` and `SCHEMA_FILE` from `bigquery_etl.schema`, `Path` from `pathlib`, and setting `schema_file_path = Path(__file__).parent / SCHEMA_FILE`.